ts/Accessibility/Options/A11yDefaults.ts

```diff
@@ -706,7 +706,7 @@ const Options: DeepPartial<A11yOptions> = {
                  *
                  * @since 8.0.0
                  */
-                skipNullPoints: true,
+                skipNullPoints: void 0,
```

Comment:
The description currently says "Skip null points when navigating through points with the keyboard". But we should also mention that the default is the opposite of the `nullInteraction` setting.